packages/types/src/create/types.ts

```diff
@@ -34,6 +34,10 @@ export interface TypeDefExtEnumDiscriminant {
   discriminant: number;
 }
 
+export interface TypeDefExtStructAlias {
```

Comment:
As above, looks fine since it is in https://github.com/polkadot-js/api/pull/1662, but not sure how it made it back with the type-on after the revert?